scripts/zombienet/polimec-polkadot-assethub-penpal.toml

```diff
@@ -0,0 +1,111 @@
+[settings]
+timeout = 1000
+provider = "native"
+
+[relaychain]
+default_command = "polkadot"
+chain = "polkadot-local"
+
+
+    [[relaychain.nodes]]
+    name = "alice"
+    ws_port = 6969
+    args = ["-lerror", "-lxcm::process_instruction=trace", "-lxcm::execute_xcm_in_credit=trace", "-lxcm::barriers=trace", "-lpallet_funding::hrmp=trace", "-lpallet_funding::test=trace", "--execution=native", ]
+    [[relaychain.nodes]]
+    name = "bob"
+
+    [[relaychain.nodes]]
+    name = "charlie"
+
+    [[relaychain.nodes]]
+    name = "dave"
+
+    [[relaychain.nodes]]
+    name = "eve"
+
+    [[relaychain.nodes]]
+    name = "ferdie"
+
+[[parachains]]
+id = 1000
+chain = "asset-hub-rococo-dev"
+
+	[[parachains.collators]]
+	name = "statemint-dev"
+	command = "../polkadot-sdk/target/release/polkadot-parachain"
+    args = ["-lerror", "-lxcm::process_instruction=trace", "-lxcm::execute_xcm_in_credit=trace", "-lxcm::barriers=trace", "-lpallet_funding::hrmp=trace", "-lpallet_funding::test=trace", "--execution=native", ]
+    ws_port = 7777
+
+
+[[parachains]]
+id = 3344
+chain = "polimec-polkadot-local"
+    [[parachains.collators]]
+    name = "collator1"
+    command = "./target/release/polimec-node"
+    args = ["--offchain-worker when-authority", "-lerror", "-lxcm::process_instruction=trace", "-lxcm::execute_xcm_in_credit=trace", "-lxcm::barriers=trace", "-lpallet_funding::hrmp=trace", "-lpallet_funding::test=trace", "--execution=native"]
```

Comment:
Maybe only use the args for 1 collator? We will probably only check the logs with a single collator. Let me know if you think otherwise